resources/web/wwi/NodeSelectorWindow.js

```diff
@@ -335,7 +335,23 @@ export default class NodeSelectorWindow {
   }
 
   getSlotType() {
-    const parentNode = this.parameter.aliasLinks[0].node;
+    let parentNode = this.parameter.aliasLinks[0].node;
+    let name = this.parameter.aliasLinks[0].name;
+    while (parentNode.isProto) {
+      let found = false;
+      for (const parameter of parentNode.parameters) {
+        if (parameter[0] === name) {
+          found = true;
+          parentNode = parameter[1].aliasLinks[0].node;
+          name = parameter[1].aliasLinks[0].name;
+          break;
+        }
+      }
```

Comment:
do we need to iterate here? Since parameters is a map (parameterName -> parameter), can't we rely on `has/get` instead?

Reply:
Good catch!